com.unity.ml-agents/Runtime/Sensors/VectorSensor.cs

```diff
@@ -172,6 +174,18 @@ public void AddObservation(IEnumerable<float> observation)
             }
         }
 
+        /// <summary>
+        /// Adds a list or array of float observations to the vector observations of the agent.
+        /// </summary>
+        /// <param name="observation">Observation.</param>
+        public void AddObservation(IList<float> observation)
```

Comment:
What are the overload resolution rules in C# for this? If the caller passes an List<float> or float[], how do I know whether the IList<float> or IEnumerable<float> will be chosen? I can give this a different name to make it less ambiguous...